src/main/java/com/jcloisterzone/board/Location.java

```diff
@@ -197,17 +194,17 @@ public Location prev() {
     }
 
     /**
-     * Returns opposite location, mirrored by axes.
+     * Returns an instance with the same mask as {@code this} but mirrored.
```

Comment:
I think this is confusing, because mirrored Location don't have same mask. You probably mean the same kind mask in meaning road/city vs farm. I would rather don't use word same.

Reply:
Would "Returns an instance that is the mirror of {@code this}" do?